docs/documentation/Logs.md

```diff
@@ -19,12 +19,12 @@ locates at ${storm.log.dir}/logs/workers-artifacts.
 Daemon and worker logs are allowed to view and download through Storm UI by authorized users.
 
 To improve the debugging of Storm, we provide the Log Search feature.
-Log Search supports searching in a certain log file or in a topology's all log files:
+Log Search supports searching in a certain log file or in all of a topology's log files:
 
-String search in a log file: In the log page for a worker, a user can search a certain string, e.g., "Exception", in for a certain worker log. This search can happen for both normal text log or rolled ziplog files. In the results, the offset and matched lines will be displayed.
+String search in a log file: In the log page for a worker, a user can search a certain string, e.g., "Exception", in for a certain worker log. This search can happen for both normal text log or rolled zip log files. In the results, the offset and matched lines will be displayed.
```

Comment:
remove "for" in "in for a certain worker log"

log->logs in "This search can happen for both normal text log"